trinity/chains/__init__.py

```diff
@@ -162,6 +163,9 @@ class DBManager(BaseManager):
     manager = DBManager(address=chain_config.database_ipc_path)  # type: ignore
     server = manager.get_server()  # type: ignore
 
+    logger = logging.getLogger('trinity.db')
+    logger.info('database: %s', os.path.abspath(chain_config.database_ipc_path))
```

Comment:
Maybe this is a debug level log? I don't think we should hint for users to direct-connect to the db.

Reply:
Doh, this was meant to be the `jsonrpc_ipc_path`